app/views/users/_filter_by_creation_dates_button.html.erb

```diff
@@ -1,7 +1,7 @@
-<div class="mb-1">
+<div class="me-2">
   <div class="w-100">
     <%= link_to(new_structure_creation_dates_filtering_path(url_params), data: { turbo_frame: 'remote_modal' }) do %>
-      <button class="btn btn-grey w-100" id="rdvi_index-nav_filter-creation-date">
+      <button class="btn btn-grey w-100 dropdown-toggle <% if params[:creation_date_after].present? || params[:creation_date_before].present? %>border-blue<% end %>" id="rdvi_index-nav_filter-creation-date">
```

Comment:
Pareil ici pour l'homogénéité je préfèrerais écrire le if statement après l'expression, mais comme c'est aussi une préférence subjective je  ping @Holist pour avoir son avis